sys-apps/ndctl/ndctl-64.1.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools bash-completion-r1 linux-info systemd
+
+DESCRIPTION="Utility library for managing the libnvdimm sub-system in the Linux kernel"
+
+HOMEPAGE="http://pmem.io/"
+SRC_URI="https://github.com/pmem/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="LGPL-2.1"
+
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RESTRICT="test"
+
+RDEPEND="dev-libs/json-c
+	sys-apps/keyutils
+	sys-apps/kmod
+	virtual/udev
+	sys-apps/util-linux"
+
+DEPEND="${RDEPEND}
+	app-text/asciidoc
+	app-text/xmlto"
+
+CONFIG_CHECK="~BLK_DEV_RAM_DAX ~FS_DAX ~X86_PMEM_LEGACY ~LIBNVDIMM ~BLK_DEV_PMEM \
+~ARCH_HAS_PMEM_API ~TRANSPARENT_HUGEPAGE ~MEMORY_HOTPLUG ~MEMORY_HOTREMOVE \
+~ZONE_DEVICE ~FS_DAX_PMD"
+
+src_prepare() {
+	test -f "${S}/version" || echo "${PV}" > "${S}/version"
+	"${S}"/git-version-gen
+	eautoreconf
+	eapply_user
```

Comment:
You can call default in the beginning of src_prepare() and leave out eapply_user (as default does it already).

Reply:
Done